src/app/utils/testing.utils.ts

```diff
@@ -397,3 +399,14 @@ function mockAxleSpacing(args?: Partial<AxleSpacing>): AxleSpacing {
 
   return {...mock, ...args};
 }
+
```

Comment:
Use this `function mockDimensions(args?: Partial<Dimensions>): Dimensions {
  const mock: Dimensions = {
    length: 100,
    width: 100,
    height: 200
  } as Dimensions;

  return { ...mock, ...args };
}`

compose this `axleSpacing: [mockAxleSpacing()]` in your test if need be. But you dont need it

Reply:
done